sdk/attestation/Azure.Security.Attestation/tests/PolicyGetSetTests.cs

```diff
@@ -0,0 +1,431 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+using Azure.Security.Attestation.Models;
+using Azure.Identity;
+using Azure.Security.Attestation.Tests.Samples;
+using System.Security.Cryptography;
+using System.Text;
+using System.Security.Cryptography.X509Certificates;
+
+namespace Azure.Security.Attestation.Tests
+{
+    public class PolicyGetSetTests : RecordedTestBase<AttestationClientTestEnvironment>
+    {
+        public PolicyGetSetTests(bool isAsync) : base(isAsync)
+        {
+        }
+
+        [RecordedTest]
+        public async Task GetPolicyShared()
+        {
+            var adminclient = GetSharedAdministrationClient();
+
+            var policyResult = await adminclient.GetPolicyAsync(AttestationType.SgxEnclave);
+            var result = policyResult.Value.AttestationPolicy;
+
+            var policyRaw = Base64Url.Decode(result);
+            var policy = System.Text.Encoding.UTF8.GetString(policyRaw);
+            Assert.IsTrue(policy.StartsWith("version"));
+
+        }
+
+        [RecordedTest]
+        public async Task GetPolicyAad()
+        {
+            var adminclient = GetAadAdministrationClient();
+
+            var policyResult = await adminclient.GetPolicyAsync(AttestationType.SgxEnclave);
+            var result = policyResult.Value.AttestationPolicy;
+
+            var policyRaw = Base64Url.Decode(result);
+            var policy = System.Text.Encoding.UTF8.GetString(policyRaw);
+            Assert.IsTrue(policy.StartsWith("version"));
+
+        }
+
+        [RecordedTest]
+        public async Task GetPolicyIsolated()
+        {
+            var adminclient = GetIsolatedAdministrationClient();
+
+            var policyResult = await adminclient.GetPolicyAsync(AttestationType.SgxEnclave);
```

Comment:
any reason not to leverage the implicit cast of `Response<T>` and  do:
```c#
StoredAttestationPolicy policyResult = await adminclient.GetPolicyAsync(AttestationType.SgxEnclave);
```

Reply:
What implicit cast? I don't think that AttestationResponse has an implicit cast.

Reply:
Ok, figured it out and changed.